_projects/2022/100385586/100385586.Rmd

```diff
@@ -10,28 +10,23 @@ output:
  distill::distill_article:
    self_contained: false
    toc: true 
----
-
-## The original graph 
-
-The [graph to replicate](https://www.pewresearch.org/religion/interactives/restrictions-on-religion-among-the-25-most-populous-countries-2007-2020/) was originally created by the Pew Research Center. 
-It is an interactive graph depicting the evolution in time of two measures related to religious conflicts: _Goverment Restrictions Index_ (GRI) and _Social Hostilities Index_ (SHI). The 25 most populous countries are included in the graph. Our goal: to replicate it as closely as possible. 
-
-
-## Set up and libraries 
 
-As with any other project, we have to load some the libraries that will be needed to complete the graph.
+---
 
 ```{r setup, fig.showtext=TRUE, echo=TRUE, message=FALSE, warning=FALSE}
 library(tidyverse)
 library(gganimate)
+library(gifski)
 library(plotly)
 library(ggtext)
 library(RColorBrewer)
+library(htmlwidgets)
+library(ggtext)
```

Comment:
I think I cleaned this up previously. I'm not sure if you need ggtext in the current version, but for sure you don't need to load htmlwidgets or gifski. Please remove them.